Sources/StreamVideo/Controllers/CallController.swift

```diff
@@ -454,6 +460,26 @@ class CallController {
         return joinCallResponse
     }
     
+    private func setupStatsTimer() {
+        statsCancellable?.cancel()
+        statsCancellable = Foundation.Timer.publish(
+            every: 5,
+            on: .main,
+            in: .default
+        )
+        .autoconnect()
+        .receive(on: RunLoop.main)
```

Comment:
Probably better to use `DispatchQueue.main` to ensure responsiveness ([reference](https://www.avanderlee.com/combine/runloop-main-vs-dispatchqueue-main/))

Reply:
good point 👍